src/pages/registration/RegistrationFormActions.tsx

```diff
@@ -166,6 +180,26 @@ export const RegistrationFormActions = ({ tabNumber, setTabNumber, validateForm
         dataTestId={dataTestId.registrationWizard.formActions.supportModal}>
         <SupportModalContent closeModal={toggleSupportModal} registration={values} />
       </Modal>
+
+      <ConfirmDialog
+        open={openNviApprovalResetDialog}
+        title={t('registration.nvi_warning.registration_is_included_in_nvi')}
+        onAccept={async () => {
+          if (tabNumber === RegistrationTab.FilesAndLicenses) {
+            setOpenNviApprovalResetDialog(false);
+            const registrationIsUpdated = await saveRegistration(values);
+            if (registrationIsUpdated) {
+              history.push(getRegistrationLandingPagePath(values.identifier));
+            }
```

Comment:
her er det dupliseringer med det som skjer i `onClickSaveAndPresent`. Får vi samlet det på en god måte?
Når bruker lagrer så ønsker vi samme flyt - uavhengig av om det er NVI-endringer eller ikke, så tror ikke vi burde trenge å implementere det som to ulike ting

(litt usikker på nøyaktig hva jeg tenker her, men vil helst unngå den type duplisering da vi helt sikkert kommer til å endre noe på en av dem en gang i fremtiden, og uten at vi da merker det vil de avvike fra hverandre, og gi en bug vi sikkert ikke kommer til å merke)